tox.ini

```diff
@@ -13,6 +13,11 @@ passenv =
 setenv =
   PYTHONPATH = .
 
+[testenv:unit]
+changedir = {toxinidir}/tests/functional
```

Comment:
Do we need the `changedir` key if all we're doing is just printing that the unit tests aren't available?